### PR TITLE
Publish npm modules to myget

### PR DESCRIPTION
**Pull request by moozzyk**

### CODE REVIEW

**Review by natemcmaster — APPROVED** [review]

File: build/repo.targets [hunk 1]
```diff
@@ -1,7 +1,8 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <NuGetPublishFeed>https://dotnet.myget.org/f/aspnetcore-ci-dev</NuGetPublishFeed>
-    <BuildDependsOn>$(BuildDependsOn);RunVerifier;RunSplitPackages</BuildDependsOn>
+    <NpmRegistry>$(NuGetPublishFeed)/npm/</NpmRegistry>
```
Comment (pranavkm) [inline build/repo.targets:4]: Nice

File: build/repo.targets [hunk 2]
```diff
@@ -55,4 +56,20 @@
...
+    <PropertyGroup>
+      <AuthTokenSetting>$(NpmRegistry.Replace("https:", "")):_authToken</AuthTokenSetting>
+    </PropertyGroup>
+    
+    <ItemGroup>
+      <NpmModule Include="$(ArtifactsDir)build\*.tgz" />
+    </ItemGroup>
+    
+    <Exec Command="npm config set $(AuthTokenSetting) $(APIKEY)" />
+    <Exec Command="npm publish --registry $(NpmRegistry) %(NpmModule.Identity)" ContinueOnError="true">
```
Comment (pranavkm) [inline build/repo.targets]: Might want to wrap it in quotes: `&quot;%(NpmModule.Identity)&quot;".

Comment (pranavkm) [reply build/repo.targets]: You might want to use `IgnoreStandardErrorWarningFormat` if the output is causing the build to fail. It'll use non-zero exit codes to fail so you can remove the `ExitCodes` check. Unless the intent is to publish as many packages as we want (which seems unusual)

**Review by pranavkm — APPROVED** [review]

File: build/repo.targets (hunk 2 above)
Comment (moozzyk) [reply build/repo.targets]: I want to revert changes to the config regardless of whether publishing packages was successful or not to not leave the api key

**Pull request merged**